### DISCUSSION TITLE
Forks fail to build with NU1903 error

### DISCUSSION THREAD

**Discussion by cornerbowlsoftware** [wixtoolset/issues#8755] [Questions] [answered]
I am able to fully build the main wix repository, however after forking, and making no changes, calling `devbuild release` ends quite a way into the build process with:

`error NU1903: Package 'WixToolset.Util.wixext' 0.0.0-build.build.1023 has a known high severity vulnerability, https://github.com/advisories/GHSA-jx4p-m4wm-vvjg`

Any ideas?

**Comment by bevanweiss** [comment, marked as answer]
You can suppress the NU1903 Error with the [WarningsNotAsErrors](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/errors-warnings) property.
Or, you can force the Nuget reference in the project in question to a version that resolves the vulnerability.  The issue can often be that the vulnerable package is only Transitive (i.e. it's not directly referenced, but is included indirectly through another package).

If you do find that incrementing a directly referenced package resolves the build error, then raising a Pull Request with that fix would be appreciated.
